sdf/1.7/pose.sdf

```diff
@@ -1,10 +1,22 @@
 <!-- Pose -->
 <element name="pose" type="pose" default="0 0 0 0 0 0" required="0">
-  <description>A position(x,y,z) and orientation(roll, pitch yaw) with respect to the frame named in the relative_to attribute.</description>
+  <description>A position(x,y,z) and orientation(roll, pitch yaw) with respect
+  to the frame named in the relative_to attribute.</description>
 
   <attribute name="relative_to" type="string" default="" required="*">
     <description>
-      Name of frame relative to which the pose is applied.
+      If specified, this pose is expressed in the named frame. The named frame
+      must be declared within the same scope (world/model) as the element that
+      has its pose specified by this tag.
+
+      If missing, the pose is expressed in the frame of the parent XML element
+      of the element that contains the pose. For exceptions to this rule and
+      more details on the default behavior, see
+      http://sdformat.org/tutorials?tut=pose_frame_semantics and
+      http://sdformat.org/tutorials?tut=pose_frame_semantics
```

Comment:
these links are now identical

Reply:
Oops, they were supposed to be pointing to the 1.5 and 1.7 versions respectively.

Reply:
Yep, you pasted versioned linkes @azeey , but the CI chain is unhappy about the fragment character (`#`) in the link and breaks if it is there. I removed them but didn't realize that it makes the link the same. 

We could patch it and remove the second link.

Reply:
See: https://github.com/ignitionrobotics/sdformat/pull/702